src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -703,6 +751,10 @@ private void HandleConflictForDocument(DocumentsOperationContext context, Replic
                         ResolveConflictToLocal(docPosition, conflictingVector);
                     }
                     break;
+                case StraightforwardConflictResolution.ResolveManually:
```

Comment:
Shouldn't be here.
The mere act of having a resolution script for the collection should cause it to be invoked, and only if it didn't resolve we'll resort to the pre-configured settings.

Note that all of that _have_ to have logging around it.